web/src/features/hfiCalculator/components/HFIPageSubHeader.tsx

```diff
@@ -0,0 +1,128 @@
+import AboutDataModal from 'features/hfiCalculator/components/AboutDataModal'
+import { HelpOutlineOutlined } from '@material-ui/icons'
+import DatePicker from 'components/DatePicker'
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    minHeight: 50,
+    marginBottom: '1rem',
+    paddingBottom: '0.5rem',
+    paddingTop: '0.5rem',
+    paddingLeft: '1rem',
+    paddingRight: '1rem',
+    fontSize: '1.3rem',
+    background: theme.palette.primary.light,
+    color: theme.palette.primary.contrastText,
+    alignContent: 'center'
+  },
+  positionStyler: {
+    position: 'absolute',
+    right: '20px'
+  },
+  helpIcon: {
+    fill: 'white'
+  },
+  aboutButtonText: {
+    color: 'white',
+    textDecoration: 'underline',
+    fontWeight: 'bold',
+    justifyContent: 'flex-end'
+  },
+  dateOfInterestPicker: {
+    marginLeft: '7px',
+    alignItems: 'center',
+    '& .MuiOutlinedInput-notchedOutline': {
+      borderColor: 'white'
+    },
+    '& .MuiOutlinedInput-input': {
+      color: 'white'
+    },
+    '& .MuiIconButton-root': {
+      color: 'white'
+    },
+    '& .MuiInputLabel-root': {
+      color: 'white'
+    },
+    '& .MuiOutlinedInput-root': {
+      borderColor: 'white'
+    },
+    '&:hover .MuiOutlinedInput-notchedOutline': {
+      borderColor: 'white'
+    },
+    '&:focus .MuiOutlinedInput-notchedOutline': {
+      borderColor: 'white'
+    }
+  }
+}))
+
+interface Props {
+  padding?: string
+  formControlClass: string
+  fireCentres: Record<string, FireCentre>
+  dateOfInterest: string
+  updateDate: (newDate: string) => void
+  selectedFireCentre: FireCentre | undefined
+  selectNewFireCentre: (newSelection: FireCentre | undefined) => void
+}
+
+export const HFIPageSubHeader: React.FunctionComponent<Props> = (props: Props) => {
+  const classes = useStyles(props)
+
+  const [modalOpen, setModalOpen] = useState<boolean>(false)
+
+  const openAboutModal = () => {
+    setModalOpen(true)
+  }
+
+  return (
+    <div className={classes.root}>
+      <Grid container spacing={1}>
+        <Grid item md={3} alignItems="center">
+          <FormControl className={classes.dateOfInterestPicker}>
+            <DatePicker
+              date={props.dateOfInterest}
+              updateDate={props.updateDate}
+              size={'small'}
+            />
+          </FormControl>
+        </Grid>
+        <Grid item md={3} alignItems="center">
+          <FormControl>
+            <FireCentreDropdown
+              fireCentres={props.fireCentres}
+              selectedValue={
+                isUndefined(props.selectedFireCentre)
+                  ? null
+                  : { name: props.selectedFireCentre?.name }
+              }
+              onChange={props.selectNewFireCentre}
+            />
+          </FormControl>
+        </Grid>
+        <Grid item md={4}>
+          {/* empty Grid item for spacing */}
+        </Grid>
+
+        <Grid item md={2} justifyContent="flex-end" alignItems="center">
```

Comment:
I think you can put this in it's own flex container with `flex-direction: row-reverse` so it starts at the rightmost space of the container. Then you can delete the empty grid items above.